### PR TITLE
[Registration] Subscribe to the newsletter during registration

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: features/account/subscribing_newsletter_during_registration.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        Given I want to register a new account
+        When I specify the first name as "Saul"
+        And I specify the last name as "Goodman"
+        And I specify the email as "goodman@gmail.com"
+        And I specify the password as "heisenberg"
+        And I confirm this password
+        And I subscribe to the newsletter
+        And I register this account
+        Then I should be notified that new account has been successfully created
+        And I should be logged in
```
Comment (lchrusciel) [inline features/account/subscribing_newsletter_during_registration.feature]: Is there any reason for this assertion?

**Comment by pjedrzejewski** [comment]
I guess you mean during registration? ;) Please rebase and fix the commit message.

File: features/account/subscribing_newsletter_during_registration.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Given I want to register a new account
+        When I specify the first name as "Saul"
+        And I specify the last name as "Goodman"
+        And I specify the email as "goodman@gmail.com"
+        And I specify the password as "heisenberg"
+        And I confirm this password
+        And I subscribe to the newsletter
+        And I register this account
+        Then I should be notified that new account has been successfully created
+        And subscription to the newsletter should be enabled
```
Comment (michalmarcinkowski) [inline features/account/subscribing_newsletter_during_registration.feature]: `And I should be subscribed to the newsletter`?

**Comment by michalmarcinkowski** [comment]
Thanks Grzesiu! Please apply my comment in a separate PR.

**Pull request merged**